app/src/main/java/com/hedvig/app/feature/keygear/ui/createitem/CreateKeyGearItemViewModel.kt

```diff
@@ -75,10 +79,23 @@ abstract class CreateKeyGearItemViewModel : ViewModel() {
     }
 }
 
-class CreateKeyGearItemViewModelImpl : CreateKeyGearItemViewModel() {
+class CreateKeyGearItemViewModelImpl(
+    private val keyGearItemsRepository: KeyGearItemsRepository
+) : CreateKeyGearItemViewModel() {
     override val createResult = MutableLiveData<CreateKeyGearItemMutation.Data>()
 
     override fun createItem() {
-        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+        viewModelScope.launch {
```

Comment:
This is so nice! when I started with coroutines viewmodel was not a thing and it was always a bit unclear wich scope you should use! This makes everything so much clearer!

Reply:
It is indeed the best thing since sliced bread 😎